client/pages/torrent/[infoHash].js

```diff
@@ -660,6 +664,20 @@ const Torrent = ({ token, torrent = {}, userId, userRole, uid, userStats }) => {
               : [getLocaleString("torrNo")],
         }}
       />
+      <Infobox mb={5}>
```

Comment:
This InfoBox should only be shown if `torrent.poster` exists, the same as the media info box